server/rooms.ts

```diff
@@ -743,6 +738,14 @@ export abstract class BasicRoom {
 				}
 			}
 			this.settings.aliases = undefined;
+
+			if (this.battle) {
```

Comment:
This changeset has a bunch of unrelated changes in it, I can't review as-is

Reply:
The important ones should be just lib/sql.ts and test/lib/sql.js. I'll see if i can properly rebase this, though there will be some changes i can't edit from Zarel's processwrapper branch.

Reply:
Okay, removed the irrelevant ones not from Zarel's branch.